### PR TITLE
Campbell changes

### PR DESCRIPTION
**Pull request by Michch333**

### CODE REVIEW

File: VirtualShopper/src/app/interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export interface UserProfile {
```
Comment (jacobmaxplumb) [inline VirtualShopper/src/app/interfaces.ts]: I think it’s sometimes a better idea to use classes so we can new things up if needed

File: VirtualShopper/src/app/user-profile/user-profile.component.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      </mat-form-field>
+
+    <mat-form-field class="example-full-width">
+      <mat-label>Bio:</mat-label>
+      <textarea #bio matInput placeholder="I love skating and prefer a laid back fit..."></textarea>
+    </mat-form-field>
+
+    <h2>Shirt</h2>
+    <h3>Size:</h3>
+    <select #shirtSize>
```
Comment (jacobmaxplumb) [inline VirtualShopper/src/app/user-profile/user-profile.component.html]: We should use reactive form so we can use materials select

Comment (Michch333) [reply VirtualShopper/src/app/user-profile/user-profile.component.html]: This one i might need you to help on. 
Will research today but might not get time to finish

Comment (jacobmaxplumb) [reply VirtualShopper/src/app/user-profile/user-profile.component.html]: I updated this

**Review by jacobmaxplumb — APPROVED** [review]

**Pull request merged**